.github/workflows/release.yml

```diff
@@ -0,0 +1,86 @@
+# This workflow will tag, release, publish to npm, and upload to gh pages.
+name: Production Release
+
+# workflow will only run when a pull request to the release branch is closed.
+on:
+    push:
+        branches: [release]
```

Comment:
Does this mean that instead of a `master`, we have a branch that matches the name of the release? Is that the idea?

Reply:
Yep that's exactly what it is!

Reply:
Do these branches exist in perpetuity in addition to the tagged releases?

Reply:
Oh actually I think this is a little different. We won't have a `master` branch, but instead have a single `release` branch, which is the currently released code. The difference between calling it `master` or `release` is just semantic. There _won't_ be a separate branch for every release (like `release-v0.6.5`), which is what I think you were referring to. The purpose of marking/saving the state of the code when a release was made is what git tag is for. So if we ever needed to go back and branch off from an old release, we would simply start from the tag for that release. 

Long story short, we'll have:
- `release`
- `development` (default)
- `feature-branches...`

Reply:
Ahh, that makes sense. Thanks for the explanation Kristo!